exdir/core/exdir_object.py

```diff
@@ -168,31 +171,31 @@ def __init__(self, root_directory, parent_path, object_name, io_mode=None,
         self.io_mode = io_mode
         self.plugin_manager = plugin_manager
 
-        validate_name = validate_name or filename_validation.thorough
-
-        if isinstance(validate_name, str):
-            if validate_name == 'simple':
-                validate_name = filename_validation.thorough
-            elif validate_name == 'thorough':
-                validate_name = filename_validation.thorough
-            elif validate_name == 'strict':
-                validate_name = filename_validation.strict
-            elif validate_name == 'none':
-                validate_name = filename_validation.none
+        name_validation = name_validation or validation.thorough
+
+        if isinstance(name_validation, str):
+            if name_validation == 'simple':
+                name_validation = validation.thorough
```

Comment:
We should perhaps remove the simple option, since we have no simple name validation yet, and only add it once it is implemented?

Reply:
I agree. We can do so in a later commit.